lib/pymedphys/tests/dicom/test_connect.py

```diff
@@ -295,3 +297,131 @@ def test_dicom_listener_cli(test_dataset):
         file_path = _build_hierarchical_path_to_plan(test_directory, test_dataset)
         read_dataset = pydicom.read_file(file_path)
         assert read_dataset.SeriesInstanceUID == test_dataset.SeriesInstanceUID
+
+
+@pytest.mark.pydicom
+def test_dicom_sender(test_dataset):
+    """Test sending DICOM objects using the DicomSender"""
+
+    scp_ae_title = "PYMEDPHYSTEST"
+
+    with tempfile.TemporaryDirectory() as tmp_directory:
+
+        test_directory = pathlib.Path(tmp_directory)
+        receive_directory = test_directory.joinpath("receive")
+        receive_directory.mkdir()
+
+        listener_command = [
+            pmp_test_utils.get_executable_even_when_embedded(),
+            "-m",
+            "pymedphys",
+            "dicom",
+            "listen",
+            str(TEST_PORT),
+            "-d",
+            str(receive_directory),
+            "-a",
+            str(scp_ae_title),
+        ]
+
+        with process(listener_command):
+            time.sleep(1)
+            dicom_sender = DicomSender(
+                host="127.0.0.1", port=TEST_PORT, ae_title=scp_ae_title
+            )
+            assert dicom_sender.verify()
+            dicom_sender.send([test_dataset])
+
+        dcm_files = receive_directory.glob("**/*.dcm")
+        dcm_file = next(iter(dcm_files))
```

Comment:
Is the `iter` needed here?

```suggestion
        dcm_file = next(dcm_files)
```